### ISSUE TITLE
Railway deploy with “No Deploys for this service”

### ISSUE THREAD

**Issue by birdfly** [nightscout/cgm-remote-monitor#7548]
I tried to create new project in Railway directly ,then choose "[cgm-remote-monitor]" ,and add variables (just copy all variables from Heroku) ,then it deployed automatically. maybe my wrong understanding made mistakes.

So this issue can be closed .

--------------------------------------------------------------------------------------------------------------------------
Hello I just follow the “https://nightscout.github.io/vendors/railway.app/new_user/?fbclid=IwAR3ZVYASvZ49jwi_G89MhqzWMFLc8oByMAXuWiAdYmyyhRuL9XAea9AzEBE” to deploy NS on railway, I configured all  Variables as recommended, after I  click deploy, it said "No Deploys for this service" show as the pics below :

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/4321387/190835412-c2a09c36-11ac-4a16-95ee-6a6f071c3375.png">

<img width="631" alt="image" src="https://user-images.githubusercontent.com/4321387/190835468-4ded20d3-7d0e-4a2a-b39d-e3bd2182c1d1.png">

**Comment by psonnera** [comment]
Yes, I need to check it again as there seems to be a issue. Just reconnect your GitHub account to continue. https://nightscout.github.io/troubleshoot/railway/#reconnect-github-to-your-project

**Comment by psonnera** [comment]
@birdfly can you help me pinpoint it, which step of the guide did you reach when you saw this message? Step 4.a?

**Comment by birdfly** [comment]
> @birdfly can you help me pinpoint it, which step of the guide did you reach when you saw this message? Step 4.a?

Yes, 4a.

I  followed first 3 steps , but step 3 is a little confusing for me . I wanted to reuse your existing MongDB Atlas database migrate from Heroku , in fact I did nothing in this steps 3 ,just copied MONGO_CONNECTION, then pasted it in step 4 after I clicked deploy in Railway. I thought I configured wrong info for MONGO_CONNECTION in Railway for there are two MONGO related variables : MONGO_CONNECTION and MONGO_TEMP. After I finished all variables and clicked deploy, it said that "No Deploys for this service".

**Comment by psonnera** [comment]
Reproduced the error this morning. Something changed as it was working before. There are now missing authorizations requests when you deploy from Github. I need to modify the sequence and will try to improve readability. MONGO_TEMP can be ignored.
